resources/guides/parking_guide.md

```diff
@@ -0,0 +1,62 @@
+---
+id: parking_guide
+title: Parking Guide
+sidebar_label: Parking Guide
+slug: /guides/parking
+---
+
+_Last updated on 2022/06/10_
+
+This is a guide to the parking spaces around the campus. See this [map](https://maps.mapsted.com/uwindsor) if you would like to see the different UWindsor buildings beforehand.
+
+## Official UWindsor Parking Spaces
+
+There are 17 official parking lots within the UWindsor campus. This includes visitor parking, staff parking, and student parking. You can locate these parking lots in this [campus map](https://web2.uwindsor.ca/pac/campusmap/index.php).
+
+![img](../../static/img/student-parking.png)
+
+### Student Designated Parking
+
+Student parkings in UWindsor are either pay-and-display, permit-only, or both. Most student parkings are located around Alumni Hall, with a few more parking locations in Leddy, Cartier Hall, Assumption Hall, and the downtown campus. Note that while some of these locations (such as the Parking Garage) do not accept cash, some other parking locations (like Leddy special night parking) are coin entry only. For places where student permits and pay-and-display options are both allowed, you do not need to pay if you already have a student permit, but you must display your permit on your vehicle when parking. The hourly rate for parking is $2.00/hr, up to $10.00 maximum per day.
```

Comment:
the part after `Note` may be better as a [docusaurus admonition](https://docusaurus.io/docs/markdown-features/admonitions)

This goes for all Notes in this PR